release/readme.html

```diff
@@ -380,13 +389,13 @@ <h2><a id="Platforms_90"></a>Platforms</h2>
         not testing on Windows 7 or 8, only 10. However, both our 32-bit and 64-bit packages now include the new
```

Comment:
Windows 11?

Reply:
@mjwitte thanks, I've updated the readme with these notes.  I'll put those changes in #9349 ahead of merging it once it's been tested.